extensions/amp-story/1.0/test/test-amp-story-share.js

```diff
@@ -2,24 +2,20 @@ import {expect} from 'chai';
 
 import {Services} from '#service';
 
-import * as analyticsApi from '#utils/analytics';
-
 import {registerServiceBuilder} from '../../../../src/service-helpers';
 import {AmpStoryShare} from '../amp-story-share';
 import {
   Action,
   AmpStoryStoreService,
   StateProperty,
 } from '../amp-story-store-service';
-import {StoryAnalyticsEvent, getAnalyticsService} from '../story-analytics';
```

Comment:
Is the removal of analytics from `test-amp-story-share.js` relevant to the PLP implementation?

Reply:
It was unrelated. The commit history was messed up. Sorry for the confusion.